src/main/kotlin/blackjack/Player.kt

```diff
@@ -0,0 +1,16 @@
+package blackjack
+
+class Player(playerName: String): Participant(playerName) {
+    override fun shouldHit(): Boolean {
+        val scoreHand = this.sumCards()
+        if (scoreHand < 21) {
+            println()
+            println("Would ${this.name} like to draw another card? (y for yes, n for no)")
+            val wantHit = readln().trim()
```

Comment:
The InputView is performing tasks that should be handled within the domain object. It looks like a change is needed. 
(println, readln)